test/common/local_reply/local_reply_test.cc

```diff
@@ -163,6 +163,24 @@ TEST_F(LocalReplyTest, TestMapperRewrite) {
       status_code: 401
       body:
         inline_string: "400 body text"
+    - filter:
+        status_code_filter:
+          comparison:
+            op: EQ
+            value:
+              default_value: 403
+              runtime_key: key_b
+      body:
+        inline_string: ""
+    - filter:
+        status_code_filter:
+          comparison:
+            op: EQ
+            value:
+              default_value: 404
+              runtime_key: key_b
+      body_format_override:
+        text_format: ""
```

Comment:
Isn't this removed?

Reply:
it's now deprecated, but not removed. since I removed the validation, I wanted to make sure there was a test case to cover the empty string case.

Reply:
I see. Thanks